src/components/TextField/TextField.tsx

```diff
@@ -136,6 +136,8 @@ interface NonMutuallyExclusiveProps {
   requiredIndicator?: boolean;
   /** Indicates whether or not a monospaced font should be used */
   monospaced?: boolean;
+  /** Indicates whether or not the entire input/text area text should be selected on focus */
+  selectAllTextOnFocus?: boolean;
```

Comment:
What are you thoughts on changing this to `selectTextOnFocus`? From `Select text`, I would assume all text is selected!

Reply:
OOH that's much better. I like that!